src/python_testing/TC_CGEN_2_9.py

```diff
@@ -48,18 +50,27 @@ class TC_CGEN_2_9(MatterBaseTest):
     async def remove_commissioner_fabric(self):
         commissioner: ChipDeviceCtrl.ChipDeviceController = self.default_controller
 
+        commissioner_fabric_index_on_dut = await self.read_single_attribute(
+            dev_ctrl=commissioner,
+            node_id=self.dut_node_id,
+            endpoint=ROOT_ENDPOINT_ID,
+            attribute=Clusters.OperationalCredentials.Attributes.CurrentFabricIndex)
+        logging.info(f"Commissioner's fabricIndex on DUT: {commissioner_fabric_index_on_dut}")
+
         fabrics: list[Clusters.OperationalCredentials.Structs.FabricDescriptorStruct] = await self.read_single_attribute(
             dev_ctrl=commissioner,
             node_id=self.dut_node_id,
             endpoint=ROOT_ENDPOINT_ID,
             attribute=Clusters.OperationalCredentials.Attributes.Fabrics)
+        logging.info(f"Fabrics table on DUT: {commissioner_fabric_index_on_dut}")
```

Comment:
Should this log `fabrics`?  As in, the log message does not match the value being logged.